packages/ember-metal/tests/map_test.js

```diff
@@ -468,13 +468,6 @@ QUnit.test('Map.prototype.constructor', function() {
   equal(map.constructor, Map);
 });
 
-QUnit.test('Map() without `new`', function() {
```

Comment:
Why do we need to remove this test?  Seems like it should still pass in debug builds, no?

Reply:
wouldn't it be just checking babel :P I think other es6 classes don't have such test, am I wrong ?

Reply:
Ya, but in general the other classes that we have in this repo are not attempting to polyfill the spec...